signing_service/signing_service/tests/tests.py

```diff
@@ -42,20 +51,25 @@ class TestSigningServiceRun:
     host = None
     port = None
     initial_reconnect_delay = None
+    parameters = None
 
     @pytest.fixture(autouse=True)
     def setUp(self, unused_tcp_port_factory):
         self.host = '127.0.0.1'
         self.port = unused_tcp_port_factory()
         self.initial_reconnect_delay = 2
-
-    def test_that_signing_service_should_be_instantiated_correctly_with_all_parameters(self):
-        signing_service = SigningService(
+        self.parameters = [
             self.host,
             self.port,
             self.initial_reconnect_delay,
             CONCENT_PUBLIC_KEY,
             SIGNING_SERVICE_PRIVATE_KEY,
+            TEST_ETHEREUM_PRIVATE_KEY,
+        ]
+
+    def test_that_signing_service_should_be_instantiated_correctly_with_all_parameters(self):
+        signing_service = SigningService(
+            *self.parameters
         )
```

Comment:
Please don't break one line statement to separate lines.

Reply:
done :+1:  here and below.